### PR TITLE
Blog section

### PR DESCRIPTION
**Pull request by shivah12**
Screenshot_11-7-2024_183351_localhost
Screenshot_11-7-2024_183336_localhost
Screenshot_11-7-2024_183321_localhost

Added blog section and also all blogs and fixed the changes and also added pagination

### CODE REVIEW

File: static/admin/config.yml [hunk 1]
```diff
@@ -60,8 +60,9 @@ collections:
           default: "project-page",
         }
       - { label: "Project Title", name: "title", widget: "string" }
-      - { label: "Image", name: "image", widget: "image"}
-      - { label: "Description", name: "description", widget: "text" }
```
Comment (Arioum) [inline static/admin/config.yml]: Undo this change as this effects the project-page section.

File: static/admin/config.yml [hunk 2]
```diff
@@ -84,50 +85,50 @@ collections:
...
                       ],
                   },
                 ],
             }
 
           - {
-              label: Services,
-              name: services,
-              widget: object,
+              label: "Services",
```
Comment (Arioum) [inline static/admin/config.yml]: Undo any changes here, for this file that is - 
1. Not related to your PR (Blog)
2. Quote related changes.

File: static/admin/config.yml [hunk 3]
```diff
@@ -27,21 +27,21 @@ collections:
...
       - { label: "Tags",
           name: "tags", 
           widget: "list",
           fields:
             [
-              { label: "Tag", name: "tag", widget: "string"}
+              { label: "Tag", name: "tag", widget: "string" }
             ]
         }
       - { label: "Description", name: "description", widget: "string" }
```
Comment (Arioum) [inline static/admin/config.yml]: change the widget value for the description field from `string` to `text` (refer project page for this)

File: src/pages/index.tsx [hunk 4]
```diff
@@ -131,13 +141,11 @@ export const query = graphql`
     ) {
       edges {
         node {
-          fields {
```
Comment (Arioum) [inline src/pages/index.tsx]: Undo this change

File: src/pages/index.tsx [hunk 5]
```diff
@@ -32,14 +35,18 @@ export const IndexPageTemplate = ({ indexPage, blogs, projects }: {indexPage: In
...
     <>
-      <Layout>
-          <Hero {...hero!} />
-          <Features {...services!} />
-          <Projects {...projects} />
-          <Testimonials {...testimonials!} />
-          {/* <FeaturedBlogs {...blogs}/> */}
-          <Contact {...contact!} />
-      </Layout>
+      <Navbar />
```
Comment (Arioum) [inline src/pages/index.tsx]: Undo this change, just uncomment the `FeaturedBlog` is enough

File: gatsby-node.ts [hunk 6]
```diff
@@ -51,23 +54,28 @@ export const createPages: GatsbyNode['createPages'] = async ({
...
+            fields {
+              slug
+            }
+          }
+        }
+      }
     }
   `);
 
-  if (!result.data) {
```
Comment (Arioum) [inline gatsby-node.ts:57]: You can keep this, no need to remove

**Review by Arioum — CHANGES_REQUESTED** [review]

**Review by Arioum — APPROVED** [review]

**Review by saurabh73 — APPROVED** [review]

**Pull request merged**